test/collection_test.js

```diff
@@ -174,6 +174,55 @@ describe("Collection", () => {
     });
   });
 
+  describe("#_setPlugins", () => {
+    var articles;
+
+    beforeEach(() => articles = testCollection());
+
+    it("should check that first argument is a supported type", () => {
+      try {
+        articles._setPlugins("bogus", {});
+        expect(false, "Should not reach here.");
+      } catch (err) {
```

Comment:
Also, you can use `expect` directly to check it raises, eg. `expect(fn).to.Throw(Error, /err msg regexp/)`.